Wikipedia/Code/ArticleRightAlignedImageCollectionViewCell.swift

```diff
@@ -50,6 +54,11 @@ open class ArticleRightAlignedImageCollectionViewCell: ArticleCollectionViewCell
             origin.y += saveButtonFrame.height
         }
         origin.y += margins.bottom
+        
+        if (apply) {
+            separator.frame = CGRect(x: 0, y: origin.y - 0.5, width: size.width, height: 0.5)
```

Comment:
Yeah, good catch. Should be `self.traitCollection.displayScale` though - view might not necessarily be on the main screen.